edi_voxel_account_invoice_oca/tests/test_voxel_account_invoice.py

```diff
@@ -238,7 +251,7 @@ def _get_products_data(self):
                     "UP": "750.0",
                     "Total": "1500.0",
                 },
-                "taxes": [],
+                "taxes": [{"Type": False, "Rate": "15.0"}],
```

Comment:
Why this change and the following one?

Reply:
Simply compare the result with the expected result and match the values.
It seems to assign a default value to the tax_ids field in the invoice lines if it does not have one and therefore all values change. I can leave it like this or set tax_ids: False, as you prefer.

Reply:
Yes, we should fix it and don't leave this to the "default behavior in some cases".

Reply:
Done, set to 15% tax for lines without.

Reply:
On contrary, what I say is to have it empty.